src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.sensors.CANCoder;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.ArmConstants;
+
+public class ArmSubsystem extends SubsystemBase {
+  private double m_pivotSpeed;
```

Comment:
no need to have this saved as a double, just call `m_pivotMotor.set(m_pivotSpeed);` in the method `setPivotSpeed` instead of `periodic`